extensions/vertx-web/runtime/src/main/java/io/quarkus/vertx/web/runtime/cors/CORSFilter.java

```diff
@@ -12,6 +15,8 @@
 
 public class CORSFilter implements Handler<RoutingContext> {
 
+    private static final String COMMA_SEPARATED_SPLIT_REGEX = "\\s*,\\s*";
```

Comment:
You need to compile the regexp, otherwise it will be compiled each time you use it.